lib/sisimai/reason/norelaying.rb

```diff
@@ -62,7 +62,7 @@ def true(argvs)
             return false if currreason =~ /\A(?:securityerror|systemerror|undefined)\z/
           else
             # Check the value of Diagnosic-Code: header with patterns
-            return true if match(argvs.diagnosticcode)
+            return true if match(argvs.diagnosticcode.downcase)
```

Comment:
__[RuboCop]__

Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.


<!-- Comment by [SideCI](https://sideci.com) -->